balboa-http/src/main/scala/com/socrata/balboa/server/EntityJSON.scala

```diff
@@ -0,0 +1,5 @@
+package com.socrata.balboa.server
+
+case class EntityJSON(timestamp: Long, metrics: Map[String, MetricJSON])
```

Comment:
JSON isn't quite accurate in the name. This is a class, not JSON. If it sometimes get serialized to JSON that's just a representation detail.

Isn't this class almost the same as the `Metrics` object? Why is there a new class?